seekinternational/seek-asia-style-guide
# feat(JobCard): remove onShelfSectionLinkClick and add linkType and payload to LinkComponent

Related to https://github.com/seekinternational/seek-asia-style-guide/pull/304
Trello card: https://trello.com/c/GV9tkv6q

Need to set omniture tracking on JobCard. Id and payload are added to LinkComponent.

## How to Test:
1. Go to JobCard
2. Check Custom Link, Job Title Component
3. Click JobTitle, get `jobTitleLink` from console.
4. Click CompanyLink, get `companyLink` from console.
5. Click Location, get `locationLink`
6. Check shelf links, click more, click job function link, get linkType and payload
